common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1222,6 +1222,16 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String REDIRECT_PREFIX = "msauth";
 
+        /**
+         * Prefix for AAD urls
+         */
+        public static final String AAD_URL_PREFIX = "https://login.microsoftonline.";
```

Comment:
We might not need the period in the end if we choose to validate the host part of the authority only (as opposed to using strings to validate prefixes)